test/test_dynamic_shape_models.py

```diff
@@ -85,6 +85,38 @@ def test_forward_pass_dynamic_input_compile_once(self):
     np.testing.assert_equal(met.metric_data('CompileTime')[0], 3)
     print('Test passed.')
 
+  @unittest.skip(
+      "disable it due to https://github.com/pytorch/xla/pull/4322#issuecomment-1374312614."
+  )
+  def test_backward_pass_with_dynamic_input(self):
+    num_features = 2
+    num_test_samples = 5
+    model = Feedforward(num_features, hidden_size=10).to(xla_dev)
+    criterion = torch.nn.BCELoss()
+    optimizer = torch.optim.SGD(model.parameters(), lr=1e-3)
+    optimizer.zero_grad()
+
+    # training
+    model.train()
+    x_training, y_training = self.create_dynamic_test_data(
+        num_test_samples, num_features, xla_dev)
+    y_pred = model(x_training)
+    loss = criterion(y_pred.squeeze(), y_training)
+    # Backpropagation.
+    loss.backward()
+    xm.optimizer_step(optimizer)
+    print('Finished training.')
+
+    # testing
+    model.eval()
+    with torch.no_grad():
+      x_test, y_test = self.create_dynamic_test_data(num_test_samples,
+                                                     num_features, xla_dev)
+      y_pred = model(x_test)
+      criterion(y_pred.squeeze(), y_test).item()
+      xm.mark_step()
+    print('Test passed.')
```

Comment:
remove this line

Reply:
sg. Do you mind if I remove it in the next pr?

Reply:
it is ok